src/controllers/playback/video/index.js

```diff
@@ -1340,6 +1340,12 @@ export default function (view) {
             case 'PageDown':
                 playbackManager.previousChapter(currentPlayer);
                 break;
+            case 'g':
+                SubtitleSync.prototype.decrementOffset();
```

Comment:
I haven't tested this yet. What if the user clicks the button before the subtitle overlay is created?
Maybe use optional chaining operator to be sure?
```suggestion
                subtitleSyncOverlay?.decrementOffset();
```
This will also allow us to correctly implement `SubtitleSync` class (drop the prototype calls).